ccViewer/ccViewerApplication.cpp

```diff
@@ -24,13 +24,22 @@
 #include "ccviewer.h"
 #include "ccViewerApplication.h"
 
-
+/*
+ * Initialise the ccViewerApp application
+ * this is version 1.40 alpha, it implements ccApplicationBase
+ * @param reference to argument
+ * @param argv pointer's pointer
+ * @param isCommandLine returns true if this is called from command line
+ * */
 ccViewerApplication::ccViewerApplication( int &argc, char **argv, bool isCommandLine )
 	: ccApplicationBase( argc, argv, isCommandLine, QStringLiteral( "1.40 alpha" ) )
 {
 	setApplicationName( "CloudCompareViewer" );
 }
-
+/*
+ * setViewer functions sets an instance of QMainWindow->ccViewer
```

Comment:
Not need to reuse the function name --> 'Sets an instance of QMainWindow->ccViewer'

And I'm not sure we want to remind that ccViewer is a QMainWindow here?

I would have wrote something like 'Sets the ccViewer instance associated to this application'